Content.Server/Lightning/LightningSystem.cs

```diff
@@ -20,6 +28,17 @@ public sealed class LightningSystem : SharedLightningSystem
     [Dependency] private readonly BeamSystem _beam = default!;
     [Dependency] private readonly IRobustRandom _random = default!;
     [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly TransformSystem _transform = default!;
+    [Dependency] private readonly IEntityManager _entMan = default!;
+
+    // a priority queue is required to iterate through Arcs of various depth
+    private PriorityQueue<LightningArc, int> _lightningQueue = new PriorityQueue<LightningArc, int>();
```

Comment:
Cool, didn't know this type existed
```suggestion
    private PriorityQueue<LightningArc, int> _lightningQueue = new();
```